llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -799,8 +799,13 @@ getStrideFromAddRec(const SCEVAddRecExpr *AR, const Loop *Lp, Type *AccessTy,
                     Value *Ptr, PredicatedScalarEvolution &PSE) {
```

Comment:
This function is only using Ptr for the purpose of dbg messages, so perhaps it would make sense to drop the argument entirely?

Reply:
There are some callers I think where including the Ptr value would help for the debug output , which is why I retained it even though it is not super pretty. Not sure if there's a more compact way to print it optionally